test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetInstallCommandTest.cs

```diff
@@ -854,6 +854,35 @@ public void InstallCommand_PackageSaveModeNuspecReinstall()
             }
         }
 
+        [Fact]
+        public void InstallCommand_ExcludeVersion_HigherVersionAlreadyInstalled()
+        {
+            using (var pathContext = new SimpleTestPathContext())
+            {
+                var outputDirectory = pathContext.SolutionRoot;
+                var source = pathContext.PackageSource;
+                // Arrange
+                PackageCreator.CreatePackage("testPackage", "1.2.0", source);
+                PackageCreator.CreatePackage("testPackage", "1.1.0", source);
+
+                // Act
+                var r = RunInstall(pathContext, "testPackage", 0, new []{"-ForceEnglishOutput","-OutputDirectory", outputDirectory, "-ExcludeVersion", "-Version", "1.2.0"});
+
+                // Assert
+                Assert.Equal(0, r.ExitCode);
+
+                // Act
+                var result = RunInstall(pathContext, "testPackage", 0, new []{"-ForceEnglishOutput","-OutputDirectory", outputDirectory, "-ExcludeVersion", "-Version", "1.1.0"});
+
+                var output = result.Output;
+
+                // Assert
+                var alreadyInstalledMessage = "Skipping installation of package \"testPackage.1.1.0\" because higher version \"testPackage.1.2.0\" is already installed.";
```

Comment:
Please use a resource instead. 

See https://github.com/NuGet/Home/issues/12820 for details.